fhir-json-comparator/raw-compare.js

```diff
@@ -0,0 +1,991 @@
+const fs = require('fs');
+
+// # Config section 1
+// set to true to display matching values as well as differences
+
+const displayMatches = false;
+
+// # Config section 2
+// fields that correlate to data that will not be in a GP2GP message,  
+// anything in these arrays will be stripped from the data before comparing
+
+const patientFieldsToIgnore = ["resource.meta", "resource.extension", "resource.identifier.extension", "resource.name", "resource.contact", "resource.gender", "resource.birthDate", "resource.address", "resource.generalPractitioner", "resource.managingOrganization"];
+const organizationFieldsToIgnore = ["resource.meta.versionId", "resource.extension", "resource.type", "resource.name", "resource.telecom", "resource.address"];
+const allergyIntoleranceFieldsToIgnore = [];
+const conditionFieldsToIgnore = [];
+const encounterFieldsToIgnore = [];
+const listFieldsToIgnore = [];
+const locationFieldsToIgnore = ["resource.meta.versionId", "resource.telecom", "resource.address", "resource.managingOrganization"];
+const medicationFieldsToIgnore = [];
+const medicationRequestFieldsToIgnore = ["resource.dispenseRequest.expectedSupplyDuration"];
+const medicationStatementFieldsToIgnore = [];
+const observationFieldsToIgnore = [];
+const practitionerFieldsToIgnore = ["resource.meta.versionId", "resource.gender"];
+const procedureRequestFieldsToIgnore = [];
+
+// # Config section 3
+// object removers remove the whole of a parent object if a condition occurs 
+// during the data stripping process
+
+const allergyIntoleranceObjectRemovers = [
+	{ "parentName": "resource.identifier", "field": "system", "checkType": "contains", "checkValues": ["https://EMISWeb", "https://PSSAdaptor"] },
+	{ "parentName": "resource.code.coding", "field": "system", "checkType": "equals", "checkValues": ["http://read.info/readv2"] },
+	//{"parentName": "resource.code.coding.extension.extension", "field": "url", "checkType" : "equals", "checkValues" : ["descriptionDisplay"] },
```

Comment:
Possibly remove commented out stuff if not needed anymore? Or is it there for a reason?

Reply:
Otherwise looks fine